ui/css/itcss/components/network.scss

```diff
@@ -128,6 +128,11 @@
   width: 100%;
 }
 
+.network-dropdown-list {
```

Comment:
It looks like based on your screencap that this div butts up against the Add Network button at the bottom. Is there any way to add some spacing so that it is less smushed?

Reply:
good call

Reply:
done in 874311f3f

Reply:
![Screenshot from 2021-11-15 15-27-05](https://user-images.githubusercontent.com/7499938/141838619-63e3f7bf-792e-4093-9a02-355fcd77ee57.png)